dataland-backend/src/main/kotlin/org/dataland/datalandbackend/service/DataManager.kt

```diff
@@ -61,8 +62,7 @@ class DataManager(
 
     override fun addDataSet(storableDataSet: StorableDataSet): String {
         verifyCompanyIdExists(storableDataSet.companyId)
-
-        val dataId = edcClient.insertData(objectMapper.writeValueAsString(storableDataSet))
+        val dataId = edcClient.insertData(objectMapper.writeValueAsString(storableDataSet))["dataId"]!!
```

Comment:
Ich finde, wir sollten ordentliche Objekte zurückgeben, und nicht einfach nur eine Map. Gerade in Kotlin sind Data Classes so einfach, und da kann man dann explizit sagen, welche Felder es geben wird. Das ist dann auch type safe.

Reply:
ok